tailwind.config.js

```diff
@@ -0,0 +1,57 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+    darkMode: ["class"],
+    content: ["./index.html", "./src/**/*.{ts,tsx,js,jsx}"],
+  theme: {
+  	extend: {
+  		borderRadius: {
+  			lg: 'var(--radius)',
+  			md: 'calc(var(--radius) - 2px)',
+  			sm: 'calc(var(--radius) - 4px)'
+  		},
+  		colors: {
+  			background: 'hsl(var(--background))',
+  			foreground: 'hsl(var(--foreground))',
```

Comment:
Pokud se ti u Kalendáře neukazujou styly jako v dokumentaci, ale jenom plain html, tak máš nejspíš špatně nastavený bundlování pro tailwind, tak koukni do vite a tailwind dokumentace. Nebo ještě může být problém, že nemáš nasetované css hodnoty pro tyhle `var(--...)` variably. Shadcn máš nastavený správně.

Reply:
Bundlování jsem zkontrolovala, vše vypadá ok. Co přesně myslíš tím "nasetované css hodnoty"? Jakože tam mají být vypsané natvrdo někde hodnoty těch proměnných?

Reply:
Koukal jsem, že ty variables už v `App.css` máš, takže to nebude ono. Asi budou nějak špatně nakonfigurované cesty mezi Vite, Shadcn a Tailwindem. Zkus si to vygooglit, jestli někdo neměl podobný problém a jak to vyřešil. Já narychlo našel zkusit přidat `"@/components/**/*.{ts,tsx}"` do `tailwind.config.js > config > content`, ale jak nemám počítač tak všechny ty možnosti neozkouším.

Reply:
V podstatě všechny problémy, co budeš muset v příštích 5 letech řešit, už někdo vyřešil. Jde jenom o to tvůj problém rozbít na malý kousky a správně ho pojmenovat (podobně, jak by ho pojmenoval jiný programátor) a odpověď najdeš. Pak už to chce jen pochopit řešení (ne to jen zkopírovat) a tím se to naučíš, abys to příště už googlit nemusela.

Reply:
Zkusila sem všechno, co lidi na netu radily. Nic nezafungovalo vyjma: 

try generating an output.css and then adding that with a link in index.html

bunx tailwindcss -i ./src/index.css -o ./src/output.css --watch --minify

Jenže to asi není úplně řešení problému, nebo jo?
Každopádně teď mám problém vrátit ty nestagnuté změny, co jsem nadělala ve snaze to nějak opravit. Zkusila jsem několik příkazů a pořád tam ty změny mám.

Reply:
Ne to řešení problému není. A v consoly v chrome dev tools ti to žádný errory nehlásí? `npm run lint` taky nemá žádné problémy? Zkoušela jsi, jestli Tailwind sám o sobě funguje, když nějaký jejich className dáš do nějaké své komponenty, co teď funguje? Prostě zkusit co nejvíce izolovat ten problém, ať víte čeho se vůbec týká...

Co se týče těch git problémů, tak koukni na dokumentaci commandu `git restore`.

Reply:
Tailwind jako takový jsem nezkousela. Zkusím zítra. Ještě sem taky cetla, ze to instaluje config do js misto do ts, ale to sem bohuzel pri instalaci nevedela a ta knihovna se neda jen tak odinstalovat, abych ji nainstalovala znovu.

Restore mi právě vcera nefungoval, ale byla jsem uz hodne unavena, tak je mozne, ze jsem delala spatne nejakou banalitu.

Reply:
Ne, errory v chrome dev tools nejsou. Změny jsem nakonec vrátila pomocí UI.
Teď jdu zkoušet ty další věci.

Reply:
Zkusila jsem ještě separátně znovu nainstalovat Tailwind. Součástí instalace byl příkaz na vygenerování toho outputu, jako jsem to dělala už předtím. Když to mám skrz ten vybuilděný outpt propojené na index.html, tak to funguje, na shadcn komponentách i na mých... To by teda mohlo být už v pořádku, nebo ne?

Reply:
Tohle sice funguje, ale není ideální, abys vždy musela pouštět script navíc. Už jsi se s tím potrápila dost, tak jsem na to kouknul a otevřel PR, kde to už funguje, jak by mělo. Tak na to pak koukni a mergni, až ti to bude jasný. Ještě nejsi úplně na levelu, že by ses tímhle musela zabývat a do detailu to chápat, takže spíš jdi řešit funkcionalitu.